packages/typespec-ts/test/unit/clientFactoryGenerator.spec.ts

```diff
@@ -364,21 +364,21 @@ describe("Client Factory generation", () => {
   });
 
   describe("should handle different auth options", () => {
-    it("should not generate credential if scope is empty", async () => {
+    it("should diagnost warning if scope is empty", async () => {
```

Comment:
Before - should not generate credential

```
export default function createClient(endpoint: string, options: ClientOptions = {})
```

Now - it will generate credential

```
export default function createClient(endpoint: string, credentials: TokenCredential, options: ClientOptions = {})
```